ci-operator/step-registry/openstack/test/e2e/openstack-test-e2e-chain.yaml

```diff
@@ -1,6 +1,8 @@
 chain:
   as: openstack-test-e2e
   steps:
+  - ref: openstack-test-e2e-images
+  - ref: mirror-images-qe-test-images
   - ref: openshift-e2e-test
   - ref: openstack-test-openstack
```

Comment:
I'm not seeing the output of the openstack specific tests in the artifacts, is it really running?

Reply:
probably not, because `openshift-e2e-test` failed. So it ended the job.
We can tackle it later though